gamemodes/terrortown/gamemode/client/cl_lang.lua

```diff
@@ -167,12 +167,29 @@ function LANG.GetUnsafeLanguageTable()
 end
 
 ---
--- Returns a translated @{string} text (if possible) directly from the language table
--- @param string name string key identifier for the translated @{string}
--- @return nil|string
+-- Returns the reference to a language table if it exists
+-- @param string name string key identifier for the language
+-- @return nil|table
+-- @realm client
+function LANG.GetUnsafeNamed(lang_name)
+	lang_name = lang_name and string.lower(lang_name)
+
+	return LANG.Strings[lang_name]
+end
+
+---
+-- Returns the reference to a language table if it exists, creates a new language if it did not exist
+-- @param string name string key identifier for the language
+-- @return nil|table
```

Comment:
You should add whether it‘s the raw or a copied table

Reply:
"Returns the reference to a language table"

Reply:
after the `@return` as the line i’ve created the comment